tests/test_cli.py

```diff
@@ -58,26 +60,25 @@ def test_cli_extras():
     output = ['click==5.0.0', 'mock==1.3.0']
     with runner.isolated_filesystem():
         shutil.copytree(DATA, abspath(join(getcwd(), 'data/')))
-        result = runner.invoke(cli, [
```

Comment:
why is not PYPF more greedy and breaks the line already after first opening bracket?